datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -431,11 +320,29 @@ macro_rules! get_null_count_values {
 
 impl<'a> PruningStatistics for RowGroupPruningStatistics<'a> {
     fn min_values(&self, column: &Column) -> Option<ArrayRef> {
-        get_min_max_values!(self, column, min, min_bytes)
+        let field = self
+            .parquet_schema
+            .fields()
+            .find(&column.name)
+            .map(|(_idx, field)| field)?;
+
+        RowGroupStatisticsConverter::new(field)
```

Comment:
There is a slight mismatch here as parquet handles schema nesting differently from arrow

I'm not sure how `Column` addresses nested fields, but I would expect to see something walking SchemaDescriptor to compute this mapping, or something similar.

Reply:
TLDR is that [Column](https://docs.rs/datafusion/latest/datafusion/common/struct.Column.html) does not address nested fields. The structure that does is 

[datafusion_physical_expr](https://docs.rs/datafusion-physical-expr/33.0.0/datafusion_physical_expr/index.html)::[expressions](https://docs.rs/datafusion-physical-expr/33.0.0/datafusion_physical_expr/expressions/index.html)::[GetFieldAccessExpr](https://docs.rs/datafusion-physical-expr/33.0.0/datafusion_physical_expr/expressions/enum.GetFieldAccessExpr.html#)

or 

https://docs.rs/datafusion/latest/datafusion/logical_expr/expr/enum.Expr.html#variant.GetIndexedField